### PR TITLE
ELV-145 add setting to reenable CPU and RAM breakdown

### LINKED ISSUE #1883: fmtQueryCPUModePct is costly, consider adding label matchers
**Describe the bug**

This query is being consistently executed: https://github.com/opencost/opencost/blob/55d77f1de56d47ecadd6f3bd38c76e8fb9a4136a/pkg/costmodel/cluster.go#L896. Because it selects `node_cpu_seconds_total` from _all_ servers, it means that it leads to huge load on the metrics stack. Consider adding selectors such as `{cluster="..."}` or something else to make the load smaller.

**To Reproduce**

1. Run OpenCost

**Expected behavior**

OpenCost not pulling in a bunch of data from unrelated servers

**Screenshots**

N/A

**Which version of OpenCost are you using?**

N/A

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by nikovacevic**
## What does this PR change?
* We are disabling costly Prometheus queries for CPU and RAM mode breakdown. This change allows those features to be re-enabled by helm configuration.

## Does this PR rely on any other PRs?
- https://github.com/opencost/opencost/pull/2049

## How does this PR impact users? (This is the kind of thing that goes in release notes!)
* Less pressure on Prometheus

## Links to Issues or ZD tickets this PR addresses or fixes
* Closes https://github.com/opencost/opencost/issues/1883
* Closes https://kubecost.atlassian.net/browse/ELV-145

## How was this PR tested?
By default:
```
2023-07-19T20:57:12.324097149Z INF ELV-145: skipping node breakdown queries
2023-07-19T20:57:12.334288408Z INF ELV-145: skipping node breakdown queries
2023-07-19T20:57:12.462450038Z INF ELV-145: skipping node breakdown queries
```
Screenshot from 2023-07-19 14-58-24

With the new setting enabled:
```
2023-07-19T21:03:06.363100429Z INF ELV-145: running node breakdown queries
2023-07-19T21:03:06.554459558Z INF ELV-145: running node breakdown queries
2023-07-19T21:03:06.749878635Z INF ELV-145: running node breakdown queries
```
Screenshot from 2023-07-19 15-07-59

## Have you made an update to documentation?
No, but I will.

### CODE REVIEW

**Review by avrodrigues5 — APPROVED** [review]
LGTM

**Comment by thomasvn** [comment]
@nikovacevic @avrodrigues5 @jessegoodier If ok with you, let's wait for a Docs PR to be approved before we merge this? Perhaps mention this config and its example output on the [`assets-api.md`](https://github.com/kubecost/docs/blob/main/assets-api.md) page?

**Comment by nikovacevic** [comment]
That's fine with me. @bstuder99 do you have thoughts? Where does this documentation live?

**Comment by brstuder** [comment]
I'm fine to go with Thomas' suggestion, but want a little more clarity on what is occurring here:
* Does this only apply to Assets queries, not Allocation/CC?
* What does the Helm flag to re-enable these queries look like?

**Comment by nikovacevic** [comment]
Good questions @bstuder99
1. Just Assets, and just Nodes and Disks within Assets. Nothing in Allocation.
2. `.Values.kubecostModel.assetModeBreakdownEnabled = true` will re-enable the feature

Also important to note that this is irreversible! Users will not be able to roll back to a pre-1.106 image once this is rolled out, or else they will panic. That's something I found out recently haha. cc @thomasvn @teevans @AjayTripathy -- in case that changes anyone's mind on whether or not this is a good idea.

**Comment by teevans** [comment]
@AjayTripathy - If you can review pretty plz.

**Comment by chipzoller** [comment]
This change does not include a placeholder setting in any values file that I can find corresponding to `kubecostModel.assetModeBreakdownEnabled`. I think this would be a requirement to accept this change.

**Comment by nikovacevic** [comment]
No review activity, and user did not accept the proposal. Just going to close.

**Pull request closed without merging**
